source/plugin/Assets/GoogleMobileAds/Platforms/Unity/DummyAdBehaviour

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEngine;
+using UnityEngine.UI;
+using GoogleMobileAds.Api;
+
+
+public class DummyAdBehaviour : MonoBehaviour
+{
+
+    public void AnchorAd(GameObject dummyAd, AdPosition position)
+    {
+        if (dummyAd != null) {
+            Image myImage = dummyAd.GetComponentInChildren<Image>();
+            RectTransform rect = myImage.GetComponent<RectTransform>();
+
+            float x = rect.sizeDelta.x/2;
```

Comment:
Will you lose precision here by casting to an int? Not sure if C++ does this but in Java I believe you'd lose precision and this would be rounded down since 2 != 2.0.

Reply:
I didn't catch that it was being casted to an int, was trying to keep everything a float. Thanks I'll change it!